pkg/cmd/server/origin/controller.go

```diff
@@ -7,6 +7,23 @@ import (
 	"github.com/openshift/origin/pkg/cmd/server/origin/controller"
 )
 
+func (c *MasterConfig) NewOpenShiftControllerPreInitializers() (map[string]controller.InitFunc, error) {
```

Comment:
yeah, not sure about the serviceaccount controller (and pullsecrets/etc.) they ran first in old world.